docs/develop/power-automate-integration.md

```diff
@@ -35,12 +35,26 @@ This opens a task pane with several options to begin connecting your Office Scri
 > [!TIP]
 > You can also start making a flow from the **More options (…)** menu on an individual script.
 
-## Excel Online (Business) connector
+## Excel connector
 
-[Connectors](/connectors/connectors) are the bridges between Power Automate and applications. The [Excel Online (Business) connector](/connectors/excelonlinebusiness) gives your flows access to Excel workbooks. The "Run script" action lets you call any Office Script accessible through the selected workbook. You can also give your scripts input parameters so data can be provided by the flow, or have your script return information for later steps in the flow.
+The [Excel Online (Business) connector](/connectors/excelonlinebusiness) gives your flows access to Excel workbooks. There are two actions that call Office Scripts.
 
-> [!NOTE]
-> The **Run script** Power Automate action only supports scripts stored in your OneDrive. To run scripts shared in SharePoint libraries, use the **Run script from SharePoint library (Preview)** action. This action is currently in preview and is subject to change based on feedback. If you encounter any issues with this action, please report them through the **Help** > **Give Feedback** option in Power Automate.
+- **Run script**. This is the standard action to use with scripts stored in your OneDrive.
```

Comment:
More specifically, this is the action to use with scripts stored in the default location in OneDrive (Documents/Office Scripts). If you store a script in your OneDrive outside of that folder, this action can't find it.

Reply:
Good point. I'll update that to be more clear and link to the doc on default OneDrive storage.